docs/get_started/getting_started_android_cmake.md

```diff
@@ -71,8 +71,16 @@ $ cmake -G Ninja -B ../iree-build-host/ -DCMAKE_INSTALL_PREFIX=../iree-build-hos
 $ cmake --build ../iree-build-host/ --target install
 ```
 
-Debugging note: if `IREE_LLVMAOT_LINKER_PATH` is set for targeting Android then
+Debugging note: 
+  * If `IREE_LLVMAOT_LINKER_PATH` is set for targeting Android then
 the build above will fail, and you should run `unset IREE_LLVMAOT_LINKER_PATH`.
+  * If you experience the build error similar to issue [#4915](https://github.com/google/iree/issues/4915), update the cmake configuration CLI to
+```shell
+$ cmake -G Ninja -B ../iree-build-host/ \
+  -DCMAKE_C_COMPILER=clang \
+  -DCMAKE_CXX_COMPILER=clang++ \
```

Comment:
This is a bit redundant with https://google.github.io/iree/get-started/getting-started-android-cmake#set-up-host-development-environment + https://google.github.io/iree/get-started/getting-started-linux-cmake#install-a-compiler, but okay. See also https://stackoverflow.com/a/22619145 - you can also set `CC` and `CXX` environment variables.

Reply:
That makes sense. The extra flags here are for debugging the build error, in case the user doesn't complete https://google.github.io/iree/get-started/getting-started-linux-cmake#build.